generator/src/Google/Service/Generator/Method.php

```diff
@@ -0,0 +1,162 @@
+<?php
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Service\Generator;
+
+class Method
+{
+    public $id;
```

Comment:
Could we please use private accessibility by default, across the board?

Reply:
Smarty needs these to be public.

Reply:
I think you can use public getters with no problem in the templates.

Reply:
@dwsupplee PTAL